### ISSUE TITLE
Targets are not created in Target Groups 

### ISSUE THREAD

**Issue by madhu131313** [kubernetes-sigs/aws-load-balancer-controller#1599]
Used:  AWS Load Balancer Controller V2

To simply the situation I m just running one single app of 2048 game

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: "awn-dev-alb-ingress"
  labels:
    app: awn-dev-alb-ingress
  annotations:
    kubernetes.io/ingress.class: alb
    alb.ingress.kubernetes.io/scheme: internet-facing
    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:ap-south-1:x:certificate/x-0168-4218-8c21-x
    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
    alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
    alb.ingress.kubernetes.io/conditions.service-game: >
      [{"Field":"host-header","HostHeaderConfig":{"Values":["game.xxxx.com"]}}]
spec:
  rules:
    - http:
        paths:
          - path: /*
            backend:
              serviceName: ssl-redirect
              servicePort: use-annotation
          - path: /*
            backend:
              serviceName: service-game
              servicePort: 80
```

Currently I can see the following error in the aws load balancer controller logs

```
"name":"awn-dev-alb-ingress","namespace":"default","error":"Internal error occurred: failed calling webhook \"mtargetgroupbinding.elbv2.k8s.aws\": Post https://aws-load-balancer-webhook-service.kube-system.svc:443/mutate-elbv2-k8s-aws-v1beta1-targetgroupbinding?timeout=30s: no endpoints available for service \"aws-load-balancer-webhook-service\""}
```

More lines generated after modifying Ingress yaml: https://pastebin.com/b5cFrTKw

I also had this issue when creating the controller

https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/1593

Thank you

**Comment by rohanme** [comment]
Is this issue resolved?If yes, what is the solution?

**Comment by madhu131313** [comment]
It's my mistake of not installing the new aws load balancer properly, initially I haven't read the document properly and have run the steps which are not necessary too. 

https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html ( Steps 5, 7 are not necessary in general)

This might have caused some issue somewhere. I have reinstalled the aws load balancer and it's working perfectly fine. 
I have created 2 clusters this way.
